server/webhook_worker.go

```diff
@@ -56,16 +56,16 @@ func (ww webhookWorker) process(msg *webhookMessage) (err error) {
 	if err != nil {
 		return err
 	}
-
-	if _, _, err = wh.PostNotifications(ww.p, msg.InstanceID); err != nil {
-		ww.p.errorf("WebhookWorker id: %d, error posting notifications, err: %v", ww.id, err)
-	}
-
 	v := wh.(*webhook)
 	if err = v.JiraWebhook.expandIssue(ww.p, msg.InstanceID); err != nil {
 		return err
 	}
 
+	v.applyReporterNotification(v.Issue.Fields.Reporter)
+
+	if _, _, err = wh.PostNotifications(ww.p, msg.InstanceID); err != nil {
+		ww.p.errorf("WebhookWorker id: %d, error posting notifications, err: %v", ww.id, err)
+	}
```

Comment:
Can we add a newline after this block?

Reply:
I'm thinking we should add a linting rule to this for the plugin repos. Do you think you can look into this Siba?

Reply:
@mickmister this is something new for me , can we do this in a different issue . seems may be can a cross plugin task

Reply:
Since we're introducing this feature, we need a way for the user to toggle it. 1/5 it's within the scope of introducing the feature. If it's done as a separate issue, the other PR should be merged first and this one can use that functionality.